component/pyroscope/ebpf/ebpfspy/bpf/profile.bpf.c

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
We can't import GPL code into Grafana Agent.

Reply:
For reference, here's an earlier discussion on that: https://github.com/grafana/agent/pull/3953#discussion_r1231996675

Reply:
Ah, thanks! That got buried a bit in the history so I missed it